suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -55,7 +65,7 @@ export const importAccountThunk = createThunk(
                         coin,
                     },
                     { ...accountInfo },
-                    imported,
+                    true,
```

Comment:
It was actually better before :D Since there are no named params I have no idea what `true` means now.